File: AP Computer Science/Labs/Unit 02/Circle/Circle.java
//© A+ Computer Science  -  www.apluscompsci.com
//Name - Sebastian Nunez   
//Date - 09/15/18
//Class - 10th
//Lab  - Area of a Circle 

public class Circle
{
	private double radius;
	private double area;

	public void setRadius(double rad)
	{
      radius = rad;
	}

	public void calculateArea( )
	{
      area = (Math.PI * Math.pow(radius,2));
	}

	public void print( )
	{  
      System.out.printf("The area is :: %.4f\n", area);
	}
}

File: Projects/Languages/Python/Labs/Unit 02/Circle/Circle.py
import math

class Circle:
   def __init__(self,radius):
      self.radius = radius

   def calc_area(self):
      return math.pi * math.pow(self.radius,2)

   def print_result(self):
      print("The area is " + str(round(self.calc_area(),4)))


class Runner():
   run = Circle(1)

   run.calc_area()
   run.print_result()
